floof.yaml

```diff
@@ -0,0 +1,59 @@
+default:
+  - command:
+      run: cargo run --bin export_schema -- ../frontend/build/schema.graphql
+      workdir: backend
+  - concurrently:
+    - http:
+        proxy: 127.0.0.1:3080
+    - run-task: frontend
+    - run-task: backend
+
+backend:
+  - set-workdir: backend
+  - watch:
+      paths:
+        - Cargo.toml
+        - Cargo.lock
+        - src/
+      run:
+        # TODO: We only need to reexport schema if something in `src/api/` changed
+        - on-change: cargo run --bin export_schema -- ../frontend/build/schema.graphql
+        - reload:
+        - cargo run
+
+frontend:
+  - set-workdir: frontend
+  - watch:
+      paths:
+        - .babelrc.js
+        - .eslintrc.js
+        - package-lock.json
+        - package.json
+        - tsconfig.json
+        - webpack.config.js
+        - src/
+      run:
+        - npx relay-compiler
+        - npx tsc --skipLibCheck
+        - npx eslint .
+        - npx webpack --mode=development --display=errors-only
```

Comment:
I kind of want to execute some of these things in parallel. Unfortunately, all of those take quite some time and they could be running in parallel. (Except for maybe interleaved output...)

Reply:
We discussed this internally already, but let me restate my opinion here for the record: I would not do it because we might get interleaved output.

Now, if `floof` were to support some kind of buffering/linearization feature for these kinds of situation ... :wink: